seed/views/v3/label_inventories.py

```diff
@@ -128,15 +137,23 @@ def label_factory(self, inventory_type, label_id, inventory_id):
             )
 
     def add_labels(self, qs, inventory_type, inventory_ids, add_label_ids):
+        """Add labels in the add_label_ids list to inventory
+
+        :param qs: QuerySet of inventory labels to exclude
+        :param inventory_type: string
+        :param inventory_ids: list
+        :param add_label_ids: list
+        """
         added = []
         if add_label_ids:
             model = self.models[inventory_type]
             exclude = self.exclude(qs, inventory_type, add_label_ids)
-            new_inventory_labels = [
-                self.label_factory(inventory_type, label_id, pk)
-                for label_id in add_label_ids for pk in inventory_ids
-                if pk not in exclude[label_id]
-            ]
+            new_inventory_labels = []
+            for label_id in add_label_ids:
+                for pk in inventory_ids:
+                    if pk not in exclude[label_id]:
+                        new_inventory_label = self.label_factory(inventory_type, label_id, pk)
+                        new_inventory_labels.append(new_inventory_label)
```

Comment:
There should be no functional change here. Just reorganized when I was debugging a test to help better understand what was happening